mlrun/model_monitoring/stream_processing.py

```diff
@@ -367,12 +367,11 @@ def apply_process_before_parquet():
         # Write the Parquet target file, partitioned by key (endpoint_id) and time.
         def apply_parquet_target():
             graph.add_step(
-                "storey.ParquetTarget",
+                "mlrun.datastore.targets.ParquetTargetStoreyWrapper",
```

Comment:
It seems to me like this problem isn't limited to model monitoring and `ParquetTarget` (even though the ticket is).

Reply:
Can you be more specific ?

Reply:
`CSVTarget`, `StreamTarget`, `NoSqlBaseTarget`, maybe others. Just search for `storage_options` in `mlrun/datastore/targets.py`.

Reply:
Those targets are not used by the model monitoring. For the mlrun regular graphs we use CM to pass the graph, so there is no credentials leakage

Reply:
What about the possibility of using `ParquetTarget` instead of creating a new wrapper class?